qemu/tests/cfg/balloon_hotplug.cfg

```diff
@@ -35,3 +35,12 @@
         - with_shutdown_after_plug:
             pm_test_after_plug = shutdown
             check_img = yes
+        - with_migrate:
+            test_tags = "evict"
```

Comment:
Is there any reason why test only evict in migrate cases?
for other cases, we tested both enlarge and evict:
 - test_tags = "evict enlarge"

Reply:
Modified, thanks!